src/test/java/io/strimzi/kafka/bridge/http/ConsumerSubscriptionIT.java

```diff
@@ -160,6 +161,62 @@ void subscriptionConsumerDoesNotExistBecauseNotCreated(VertxTestContext context)
         context.completeNow();
     }
 
+    @Test
+    void subscriptionConsumerEmptyTopics(VertxTestContext context) throws InterruptedException, ExecutionException, TimeoutException {
+        String name = "my-kafka-consumer-empty-subscription";
+        adminClientFacade.createTopic(topic);
+
+        JsonArray topics = new JsonArray();
+        topics.add(topic);
+
+        JsonObject topicsRoot = new JsonObject();
+        topicsRoot.put("topics", topics);
+
+        // Create consumer and subscribe to the topic
+        JsonObject consumerJson = new JsonObject();
+        consumerJson.put("name", name);
+        consumerJson.put("format", "json");
+        consumerService()
+                .createConsumer(context, groupId, consumerJson)
+                .subscribeConsumer(context, groupId, name, topicsRoot);
```

Comment:
after subscribing you should verify that it was ok by listing the subscription, so that after unsubscribe you can do the same and checking it's empty.
Anyway, for how the Kafka client works, after the subscribe operation you need to execute some poll/consume in order to allow the consumer to be really subscribed (joining the consumer group). The subscribe operation itself just fill a list inside the Kafka consumer nothing more.
See the `listConsumerSubscriptions` test which does something like that after subscribe and before listing the subscriptions.